disputer/index.js

```diff
@@ -24,9 +24,10 @@ const ExpandedERC20 = artifacts.require("ExpandedERC20");
  *     mode which will exit after the loop.
  * @param {Object} priceFeedConfig Configuration to construct the price feed object.
  * @param {Object} [disputerConfig] Configuration to construct the disputer.
+ * @param {String} [disputerOverridePrice] String encoded BigNumber to override the disputer price feed.
```

Comment:
When you say `String encoded BigNumber` do you mean a String representing a Wei number?

Reply:
sure. I guess your description is more specific as it's not actually a bignumber. it's a string, representing a number, scaled by an arbitrary number of decimal points. Have refactored the comment to be more spesific.